src/jit/gentree.h

```diff
@@ -1169,6 +1169,11 @@ struct GenTree
         }
     }
 
+    // NOTE: these functions are defined in lir.h.
+    inline void SetIsUnusedValue();
```

Comment:
I would be more specific than "these", e.g., "The functions to set/clear/check the UnusedValue bit are defined in lir.h"